.travis.yml

```diff
@@ -10,7 +10,8 @@ php:
   - hhvm
 before_script:
   - composer install
+dist: trusty
```

Comment:
That would allow 5.3 to work and [drop HHVM support](https://travis-ci.org/rdlowrey/auryn/jobs/239647858#L134).